server/BraveAlerterConfigurator.js

```diff
@@ -1,144 +1,145 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
@trisapeace oh here is an example of one I was ??? about - I'm not 100% sure where it wants the `this` - I can put it literally anywhere and it stops complaining but then tests fail 🤣 So it's disabled for now

Reply:
Is it only on some of the functions? If so, they should probably just be marked `static` and that might make this go away.

Reply:
So adding static before the 3 it was complaining about* makes tests fail and a whooooole ton of unhandled promise rejection warnings happen

* `alertSessionChangedCallback`, `createIncidentCategoryKeys`, and `createResponseStringFromIncidentCategories`

Reply:
as I write that though I remember I didn't pull in changes so maybe that will help on at least the promises front

Reply:
Hmmm, for `createIncidentCategoryKeys` and `createResponseStringFromIncidentCategories`, you probably just need to change everywhere they are being called from `this.createIncidentCategoryKeys` to `BraveAlerterConfigurater.createIncidentCategoryKeys` and same for the other one.

`alertSessionChangedCallback` is trickier because it is called by the Brave Alert Lib. Might just need to do the `eslint-disable` on this one.

Reply:
Okay I will look into that - I might be a bit though because I just tried to merge in the updated master and it is very mad and wanting to change everything back to `module.exports.thing` format and I can't run tests until I fix that lol

Reply:
If I change it to be like that it complains about `this` in even more functions 🤔

Reply:
Hmmm, maybe because calling these functions was the only place that those ones used `this` as well. Here's the docs: https://eslint.org/docs/rules/class-methods-use-this . Looks like to change them to static, it does mean that the caller needs to change the way it's being called, which will mess us up because of Brave Alert Lib. In that case, I think you've done it right to just disable the rule for this file. Thanks for investigating!